Client-app/src/components/SignUp/SignUp.css

```diff
@@ -44,6 +44,14 @@ fieldset {
   color: var(--text-primary) !important;
 }
 
-.SIGN-UP__link:hover, .SIGN-IN__link:hover {
+.SIGN-UP__link:hover,
+.SIGN-IN__link:hover {
   color: var(--text-primary) !important;
-}
\ No newline at end of file
+}
+
+.css-nosrnw {
```

Comment:
This classname used both in SignUp.css & SignIn.css causes the pages to look weird. Otherwise the rest of the code looks good!